### PR TITLE
Migrate ACS to new lib location

### PR DESCRIPTION
**Pull request by elopezanaya**
# PR Details

### Description
_Include a description of the problem to be solved_

As part of security improvements ACS has been moved to a new location

This PR is based on - https://github.com/microsoft/omnichannel-chat-sdk/pull/399

## Solution Proposed
_Detail what is the solution proposed, include links to design document if required or any other document required to support the solution_

- ACS adapter was moved to a new lib location
- remove explicit defined libs for azure, which are already contained in the ACS lib

### Acceptance criteria

_Define what are the conditions to consider the PR has achieved the intended goal_

## Test cases and evidence
_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_

### Playwright 
image

### Omnichannel chat SDK 
image

### Web (Omnichannel Chat Wiidget)
image

### React Native (Android)
image

### Sanity Tests

- [ ] You have tested all changes in Popout mode
- [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
- [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y

- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

***Please provide justification if any of the validations has been skipped.***

### CODE REVIEW

File: src/telemetry/AriaTelemetry.ts [hunk 1]
```diff
@@ -805,7 +806,7 @@ class AriaTelemetry {
             Event: '',
             ExceptionDetails: '',
             ElapsedTimeInMilliseconds: '',
-            ACSChatVersion: require('@azure/communication-chat/package.json').version // eslint-disable-line @typescript-eslint/no-var-requires
+            ACSChatVersion: require('@microsoft/botframework-webchat-adapter-azure-communication-chat/node_modules/@azure/communication-chat/package.json').version // eslint-disable-line @typescript-eslint/no-var-requires
```
Comment (emlynmac) [inline src/telemetry/AriaTelemetry.ts:809]: I'm curious why this change was needed; is there a version difference between the direct include and that used in the adapter?

**Review by dleblondMSFT — APPROVED** [review]

**Review by xTEddie — APPROVED** [review]

**Review by xTEddie — APPROVED** [review]
